teamworkApp/app/controllers/student_graph_controller.rb

```diff
@@ -3,7 +3,7 @@ def show
         # @students: Stores all the student objects
         # @selected: Stores whether the student was selected previously or not
         # @image: Stores the image name of the graph (if available)
-
+        @image1 = ''
```

Comment:
Is this line necessary since you declare @image1 later on in both the if and else statement?

Reply:
You're totally right i fogot to take it out from when I was experimenting